### ISSUE TITLE
Create an API Call for /file

### ISSUE THREAD

**Issue by lomky** [USGCRP/gcis#399]
Implement the API call for "/file", like "/figure" that will produce the TTL that can then be used to load the RDF. Another ticket on USGCRP/gcis should be created and linked to this one (as a precursor).

Pull request [gcis-rdf#5](https://github.com/USGCRP/gcis-rdf/pull/5) is blocked by this feature.

**Comment by zacharylandes** [comment]
Closing due to age and irrelevance
